src/__tests__/Cell.spec.js

```diff
@@ -84,7 +87,8 @@ describe('Cell Tests', () => {
       testElement = TestUtils.renderIntoDocument(<Cell {...testProps}/>);
       var editorContainerInstance = TestUtils.findRenderedComponentWithType(testElement, EditorContainerStub);
       expect(editorContainerInstance).toBeDefined();
-      expect(editorContainerInstance.props).toEqual({
+      var expectedProps = Immutable.Map(editorContainerInstance.props).toJS();
+      expect(expectedProps).toEqual({
```

Comment:
`props` are Immutable these days, so this comparison was failing. the way `jasmine` does it's equality comparison is by attaching a new property to each object, so it was attaching to the ones we pass in but not the `editorContainerInstance` ones, and failing.

since jasmine apparently can't actually handle equating frozen objects, i decided to map the `editorContainerInstance` ones to a mutable object...? i dunno if that's right but it made sense at the time and does work... :flushed: